.github/workflows/ci.yml

```diff
@@ -482,24 +482,18 @@ jobs:
         with:
           input: apis
 
-      - name: Detect Breaking Changes in Protocol Buffers (Master Branch)
+      # buf-breaking-action doesn't support branches
+      # https://github.com/bufbuild/buf-push-action/issues/34
+      - name: Detect Breaking Changes in Protocol Buffers
         uses: bufbuild/buf-breaking-action@a074e988ee34efcd4927079e79c611f428354c01 # v1
-        # We want to run this for the master branch, and PRs.
-        if: ${{ ! startsWith(github.ref, 'refs/heads/release-') }}
+        # We want to run this for the master branch, and PRs against master.
+        if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
Looks like this isn't running for PRs against master, I'm guessing it will only run on merge.